mobile/test/common/integration/test_server.h

```diff
@@ -37,8 +40,10 @@ class TestServer {
   /**
    * Starts the server. Can only have one server active per JVM. This is blocking until the port can
    * start accepting requests.
+   * use_quic: If true, use http3 with TLS
+   * disable_https: If true: use http1 without TLS.  If false, use http2 with TLS
    */
-  void startTestServer(bool use_quic);
+  void startTestServer(bool use_quic, bool disable_https);
```

Comment:
this is kind of confusing and there aren't many calls to `startTestServer`... might be better to pass in an enum, e.g.
```
enum HttpProtocol {
  HTTP3,
  HTTP2_WITH_TLS,
  HTTP1_WITHOUT_TLS,
```

Reply:
Talked with Ali in chat.  I looked into passing enums from swift/kotlin to c++ before and it seems like a minor pain so I'd rather leave as-is since this won't get used much